gradle-plugin/build.gradle

```diff
@@ -4,6 +4,7 @@ plugins {
     id "de.fuerstenau.buildconfig" version "1.1.8"
     id "java-gradle-plugin"
     id "groovy"
+    id "maven-publish"
```

Comment:
`./gradlew publish` will also publish the gradle-plugin artifacts, which will fail to upload to Maven Central because the pom is incomplete and they are not signed... ?

Reply:
I don't think so, because the settings from `publish.gradle` are not applied to this project, so no maven repository to publish to is set.

Reply:
Good point, sounds logical!